Corrected syntax errors

esecrpm opened the pull request.

esecrpm committed:
Merge pull request #1 from mark-hallman/master

Sync with master branch

esecrpm committed:
Corrected syntax error

Replaced '*' (asterisk) with '.+' (dot-plus) in registry commands.

esecrpm committed:
Corrected syntax errors

Replaced '*' (asterisk) with '.+' (dot-plus) in registry commands
Corrected location of SchedLgU.txt

mark-hallman reviewed (commented):
Changes approved.  Thanks!

mark-hallman reviewed (approved):
Thanks for the input.  I thought I merged this weeks ago.

esecrpm committed:
Include User's DMP Files

Include individual users' process crash dumps located in AppData\Local\CrashDumps
